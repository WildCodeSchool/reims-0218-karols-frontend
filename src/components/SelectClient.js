import React from "react"
import { Label } from "reactstrap"

export default class SelectPerson extends React.Component {
  render() {
    return (
      <Label for="selectClient">
        Select person :
        <input type="number" step="1" value="0" min="0" max="14" />
      </Label>
    )
  }
}
