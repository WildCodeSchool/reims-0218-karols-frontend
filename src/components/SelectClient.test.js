import React from "react"
import ReactDOM from "react-dom"
import SelectClient from "./SelectClient"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<SelectClient />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<SelectClient />).toJSON()
  expect(tree).toMatchSnapshot()
})
