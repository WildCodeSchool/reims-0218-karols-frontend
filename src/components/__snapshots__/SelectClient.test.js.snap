// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mt-3 form-group"
>
  <label
    className=""
    htmlFor="select-client"
  >
    Select client : 
  </label>
  <input
    className="form-control"
    id="select-client"
    max="6"
    min="0"
    onChange={[Function]}
    placeholder="Indiquez le nombre de personnes"
    type="number"
  />
</div>
`;
